### PR TITLE
Pin bls-wallet-clients@0.8.1

### PR DESCRIPTION
**Pull request by jacque006**
## What is this PR doing?
Pin `bls-wallet-clients@0.8.1`.

## How can these changes be manually tested?
CI, testing covered in previous PRs

## Does this PR resolve or contribute to any issues?
No

## Checklist
- [x] I have manually tested these changes
- [x] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: aggregator-proxy/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "@types/koa__cors": "^3.3.0",
     "@types/koa__router": "^8.0.11",
     "@types/node-fetch": "^2.6.1",
-    "bls-wallet-clients": "0.8.1-758c7c4",
+    "bls-wallet-clients": "0.8.1",
```
Comment (blakecduncan) [inline aggregator-proxy/package.json:24]: I'm hesitant to pin this version yet.  I'm testing this version out in the browser wallet with the latest experimental version and I'm getting a node-fetch error.  I'm wondering if we should remove it?  @JohnGuilding did you need to add node-fetch for the provider work?

Comment (jacque006) [reply aggregator-proxy/package.json:24]: Saw that, yeah we can hold off until we resolve that fetch issue

**Comment by jacque006** [comment]
Closing in favor of https://github.com/web3well/bls-wallet/pull/432

File: aggregator-proxy/package.json (hunk 1 above)
Comment (jacque006) [reply aggregator-proxy/package.json:24]: Fixed in https://github.com/web3well/bls-wallet/pull/431

**Pull request closed without merging**